pkg/pillar/zedcloud/send.go

```diff
@@ -201,6 +223,10 @@ func VerifyAllIntf(ctx *ZedCloudContext,
 		log.Error(err.Error())
 		return false, remoteTemporaryFailure, intfStatusMap, err
 	}
+
+	ctxWork, cancel := GetContextForAllIntfFunctions(ctx)
```

Comment:
I could misremember but I thought Verify was using 15 second timeout per http POST/GET as opposed to 120 seconds. Whether we have that 15 http second timeout the context timeout can (and maybe should) be lower than 4 minutes. Maybe your automatic 3 x http timeout takes care of that but it would make sense to double check.

Reply:
Cannot find difference for verifiyAllIntfs. Actually I see that inside launchHostProbe we use maxRemoteProbeWait as timeout, but it is inside goroutine, so I put context.Background there.